.github/workflows/publish-cuda-docker-image.yml

```diff
@@ -6,11 +6,9 @@ on:
       sdk_requirement:
         type: string
         description: |
-          Orquestra SDK requirement to build the image with. 
-          Example of using a published version of the SDK:
-          orquestra-sdk[ray]==0.63.0
-          Example of using an unpublished version:
-          orquestra-sdk[ray] @ git+https://github.com/zapata-engineering/orquestra-sdk.git@v0.63.0#subdirectory=projects/orquestra-sdk
+          Orquestra SDK revision
+          You can use git tag (like "v0.62.0")
+          or branch name (like "main")
```

Comment:
**nitpick**: it's very easy to get lost with versions, refs, and requirements when dealing with our CI/CD (speaking from experience). Looks like this parameter will no longer be a pip-compliant _requirement_ but a _git ref_ instead. Can you rename the parameter?